src/pages/home/report/ReportActionsList.js

```diff
@@ -163,7 +167,7 @@ class ReportActionsList extends React.Component {
                     keyExtractor={this.keyExtractor}
                     initialRowHeight={32}
                     initialNumToRender={this.calculateInitialNumToRender()}
-                    onEndReached={this.props.loadMoreChats}
+                    onEndReached={() => !this.props.network.isOffline && this.props.loadMoreChats()}
```

Comment:
Could we not move the isOffline check into the `loadMoreChats` function itself?

Reply:
Agree, that would be cleaner.